src/Trakx.CoinGecko.ApiClient/openApi3.dev.yaml

```diff
@@ -0,0 +1,349 @@
+﻿openapi: 3.0.0
+info:
```

Comment:
this was because Shrimpy confusingly has 2 very similar APIs, one for normal and one for dev users, you can remove this file

Reply:
I will do it.

Reply:
Removed